@Djelibeybi the dependency pins unnecessary dependencies:
```
Using Python 3.13.1 environment at: venv
  × No solution found when resolving dependencies:
  ╰─▶ Because aiolifx-themes==0.6.1 depends on pytest-asyncio>=0.25.2,<0.26.0
      and you require aiolifx-themes==0.6.1, we can conclude that you require
      pytest-asyncio>=0.25.2,<0.26.0.
      And because you require pytest-asyncio==0.24.0, we can conclude that
      your requirements are unsatisfiable.
Error: Process completed with exit code 1.
``` 

It looks like the release added the test dependencies as required dependencies:
https://github.com/Djelibeybi/aiolifx-themes/commit/0ab6d863488914d1241f0f4f498a227007fb3722
https://github.com/Djelibeybi/aiolifx-themes/commit/8993979a17d9325624bd9304b5e40b03c7d15e3c

This should not be part of the release but part of dev/testing

--- a/homeassistant/components/lifx/manifest.json
+++ b/homeassistant/components/lifx/manifest.json
@@ -53,6 +53,6 @@
   "requirements": [
     "aiolifx==1.1.2",
     "aiolifx-effects==0.3.2",
-    "aiolifx-themes==0.6.1"
+    "aiolifx-themes==0.6.2"
   ]
 }

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -285,7 +285,7 @@ aiokef==0.2.16
 aiolifx-effects==0.3.2
 
 # homeassistant.components.lifx
-aiolifx-themes==0.6.1
+aiolifx-themes==0.6.2
 
 # homeassistant.components.lifx
 aiolifx==1.1.2

--- a/requirements_test_all.txt
+++ b/requirements_test_all.txt
@@ -267,7 +267,7 @@ aiokafka==0.10.0
 aiolifx-effects==0.3.2
 
 # homeassistant.components.lifx
-aiolifx-themes==0.6.1
+aiolifx-themes==0.6.2
 
 # homeassistant.components.lifx
 aiolifx==1.1.2

